docs/becoming-a-maintainer/translations/es/potenciadores-para-mantenedores.md

```diff
@@ -0,0 +1,140 @@
+
+#### Herramientas de análisis de código
+
+El análisis de código es una herramienta para detectar vulnerabilidades de seguridad, posibles bugs y errores en el código. Puedes utilizar la función de [análisis de código](https://docs.github.com/en/code-security/code-scanning) de GitHub y configurar herramientas como [CodeQL](https://docs.github.com/en/code-security/code-scanning/introduction-to-code-scanning/about-code-scanning-with-codeql), que GitHub mantiene, o herramientas de análisis de terceros como [SonarQube](https://github.com/marketplace/actions/official-sonarqube-scan).
+
+#### Pruebas
+
+Configurar acciones para ejecutar pruebas es útil para garantizar que tu aplicación funciona y se desempeña como debería y que no se producen regresiones con los nuevos cambios. [Cypress GitHub Action](https://docs.cypress.io/guides/continuous-integration/github-actions) es uno de los ejemplos de acciones para pruebas E2E. También puedes consultar [Is Website Vulnerable](https://github.com/marketplace/actions/is-website-vulnerable) para encontrar vulnerabilidades de seguridad conocidas públicamente en los sitios web de las bibliotecas de JavaScript o [Step CI Action](https://github.com/marketplace/actions/step-ci-action) para el control de calidad de la API.
+
+### Creación y personalización de acciones
+
+Puedes buscar GitHub Actions disponibles en el [GitHub Marketplace](https://github.com/marketplace?type=actions). Pero si no puedes encontrar la que necesitas, puedes crear o personalizar tus propias acciones a partir de las GitHub Actions existentes.
+
+Brian Douglas, el fundador de OpenSauced, creó [Take Action](https://github.com/marketplace/actions/contributor-takes-action). Esta acción permite a los contribuidores asignarse a sí mismos a un problema escribiendo el comando `.take` en el comentario del problema, lo que permite a los mantenedores centrarse en tareas más importantes que la asignación de problemas.
+
+Sin embargo, queremos que los contribuidores solo puedan tomar problemas que hayan pasado nuestra priorización o que no estén destinados a ser trabajados por el equipo central. Por lo tanto, creamos la [Acción de Priorización](https://github.com/open-sauced/app/blob/beta/.github/workflows/triage.yml) que bloqueará Take Action cuando exista una etiqueta `needs triage` o `core team work`.
+
+Puedes leer más sobre las GitHub Actions y cómo crear una en [la documentación oficial](https://github.com/features/actions).
+
+Aquí tienes algunos recursos adicionales para ofrecerte más información sobre las GitHub Actions y cómo configurar una:
+
+- [GitHub Actions: El mejor amigo de un mantenedor](https://dev.to/opensauced/github-actions-a-maintainers-best-friend-488n)

```

Comment:
Instead of GitHub Actions consider using Acciones de GitHub as it is part of the glossary